src/scheduler.c

```diff
@@ -6,7 +6,9 @@
  *
  *-------------------------------------------------------------------------
  */
+#ifdef GLIBC
```

Comment:
again, not sure if that's even a good idea; found a similar fix in https://github.com/PowerDNS/pdns/pull/1958/files